### PR TITLE
tweak

### PR DESCRIPTION
**Pull request by after12am**

### CODE REVIEW

File: test/a.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+a
+
+b
```
Comment (after12am) [inline test/a.js]: ok

File: test/a.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+c
+
+d
```
Comment (okami-satoshi) [inline test/a.js]: I got it.

File: test/a.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+e
+
```
Comment (okami-satoshi) [inline test/a.js]: wip

File: test/a.js (hunk 1 above)
Comment (after12am) [inline test/a.js]: no

File: test/a.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+a
+
+b
+
+c
```
Comment (after12am) [inline test/a.js]: I understood.

File: test/a.js (hunk 1 above)
Comment (after12am) [inline test/a.js]: aaaaaaaa

File: test/a.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (after12am) [inline test/a.js]: wowwwwwwww

**Pull request closed without merging**